ruby_smb.gemspec

```diff
@@ -36,4 +36,6 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'rubyntlm'
   spec.add_runtime_dependency 'windows_error'
   spec.add_runtime_dependency 'bindata'
+  spec.add_runtime_dependency 'openssl-ccm'
+  spec.add_runtime_dependency 'openssl-cmac'
```

Comment:
Context question from me, why do we need these? I'm guessing Ruby's openssl doesn't ship with this by default?

Reply:
Unfortunately, Ruby's OpenSSL does not support CMAC and there was a bug with CCM. Spencer actually [fixed](https://github.com/ruby/openssl/pull/359) the later, but we need that gem until the upstream releases a new version that will include these changes.